docs/submarine-sdk/pysubmarine/tracking.md

```diff
@@ -20,6 +20,16 @@
 - [TODO] (WEB) Metric tracking ui in workbench-web
 - [TODO] (WEB) Metric graphical display in workbench-web
 
+## Quickstart
+1. Start mini-submarine
+
```

Comment:
Can you add a `start mini-submarine` command here?

Reply:
I add a link to [mini-submarine/README.md](https://github.com/pingsutw/hadoop-submarine/tree/SUBMARINE-213/dev-support/mini-submarine#run-mini-submarine-image), so users can get more information about how to start and build mini-submarine